<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUserGUI</class>
 <widget class="QWidget" name="MainUserGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>941</width>
     <height>761</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>LEDs</string>
    </attribute>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>97</width>
       <height>54</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="rojo">
      <property name="text">
       <string>Rojo</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="verde">
      <property name="text">
       <string>Verde</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="azul">
      <property name="text">
       <string>Azul</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>171</width>
       <height>243</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QwtKnob" name="Knob">
        <property name="upperBound">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
        <property name="singleSteps">
         <UInt>0</UInt>
        </property>
        <property name="pageSteps">
         <UInt>10</UInt>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Brillo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="color_widgets::ColorWheel" name="colorWheel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="Mode_PWM_check">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>PWM Mode</string>
     </property>
    </widget>
    <widget class="Led" name="led1">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Led" name="led4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="color">
      <color>
       <red>1</red>
       <green>14</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="LeeSW_check">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>LeeSW</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ADC</string>
    </attribute>
    <widget class="QPushButton" name="ADCButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LeeADC</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>271</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLCDNumber" name="lcdCh1">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdCh3">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLCDNumber" name="lcdCh2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdCh4">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="ThermoMeter" name="thermometer">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>50</y>
       <width>71</width>
       <height>251</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Shows the pressure</string>
     </property>
     <property name="whatsThis">
      <string>The bar meter widget displays the pressure attached to it</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Otro tab</string>
    </attribute>
    <widget class="LED" name="led8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>50</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtPlot" name="qwtPlot">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>400</width>
       <height>227</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtKnob" name="frecuencia">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>P-1-3-1</string>
    </attribute>
    <widget class="QCheckBox" name="ACMGPIO4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ACMGPIO5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ACMGPIO6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ACMGPIO7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO7</string>
     </property>
    </widget>
    <widget class="Led" name="led_GPIO0">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Led" name="led_GPIO1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Led" name="led_GPIO2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Led" name="led_GPIO3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GPIO3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>P-1-3-5</string>
    </attribute>
    <widget class="QwtKnob" name="frecuencia_sample">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>460</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>5.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>20.000000000000000</double>
     </property>
    </widget>
    <widget class="QwtPlot" name="qwtPlot_acc">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>400</width>
       <height>227</height>
      </rect>
     </property>
    </widget>
    <widget class="QwtPlot" name="qwtPlot_velo">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>410</y>
       <width>400</width>
       <height>227</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>641</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="serialPortLabel">
     <property name="text">
      <string>Puerto Serie:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="serialPortComboBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pingButton">
     <property name="text">
      <string>Ping</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>691</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>Estado:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="text">
     <string>Detenido</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ThermoMeter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/thermometer.h</header>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>LED</class>
   <extends>QWidget</extends>
   <header>LED.h</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
